test-utils/src/lib.rs

```diff
@@ -95,23 +95,23 @@ macro_rules! impl_frame_system {
 			type BaseCallFilter = frame_support::traits::Everything;
 			type BlockWeights = ();
 			type BlockLength = ();
-			type AccountId = AccountId;
```

Comment:
Did it want you to sort the type assignment in the same order as the type definitions in the trait definition?

Reply:
tbh i couldnt find an issue i was looking for so i copied the whole thing from somewhere else and adjusted from there :)

Reply:
Hahaha, fine.